types/d3-color/index.d.ts

```diff
@@ -6,9 +6,10 @@
 //                 denisname <https://github.com/denisname>
 //                 Hugues Stefanski <https://github.com/ledragon>
 //                 Nathan Bierema <https://github.com/Methuselah96>
+//                 Fil <https://github.com/Fil>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-// Last module patch version validated against: 3.0.1
+// Last module patch version validated against: 3.1.0
```

Comment:
You need to bump the version number on line 1 of this file as well.

Reply:
done